src/features/Courses/AddClass/index.jsx

```diff
@@ -0,0 +1,219 @@
+      >
+        <ModalDialog.Header>
+          <ModalDialog.Title>
+            Add Class
+          </ModalDialog.Title>
+          <p className="course-name my-1">
+            {courseInfo?.masterCourseName}
+          </p>
+        </ModalDialog.Header>
+        <ModalDialog.Body>
+          <Form onSubmit={handleAddClass}>
+            <FormGroup controlId="instructorInfo">
+              <Form.Control
+                type="text"
+                placeholder="Class name / Title *"
+                floatingLabel="Class name / Title *"
+                className="my-1 mr-0"
+                name="name"
+                value={className}
+                onChange={(e) => setClassName(e.target.value)}
+                required
+              />
+            </FormGroup>
+            <Form.Group as={Col} className="px-0">
+              <Select
+                placeholder="Instructor"
+                name="instructor"
+                options={instructorsOptions}
+                onChange={option => setInstructorSelected(option)}
+                value={instructorSelected}
+              />
+            </Form.Group>
+            <Form.Row>
+              <Form.Group as={Col}>
+                <Form.Control
+                  type="date"
+                  placeholder="Start date *"
+                  floatingLabel="Start date *"
+                  className="my-1 mr-0"
+                  name="start_date"
+                  required
+                />
+              </Form.Group>
+              <Form.Group as={Col}>
+                <Form.Control
+                  type="date"
+                  placeholder="End date"
+                  floatingLabel="End date"
+                  className="my-1 mr-0"
+                  name="end_date"
+                />
+              </Form.Group>
+            </Form.Row>
+            <Form.Row>
+              <Form.Group as={Col}>
+                <Form.Control
+                  type="text"
+                  placeholder="Minimum enrollment"
+                  floatingLabel="Minimum enrollment"
+                  className="my-1 mr-0"
+                  name="min_students_allowed"
+                  value={minStudents}
+                  onChange={e => handleNumericField(e, 'min')}
+                  pattern="[0-9]*"
+                />
+              </Form.Group>
+              <Form.Group as={Col}>
+                <Form.Control
+                  type="text"
+                  placeholder="Maximum enrollment"
+                  floatingLabel="Maximum enrollment"
+                  className="my-1 mr-0"
+                  name="max_students_enrollments"
+                  value={maxStudents}
+                  onChange={e => handleNumericField(e, 'max')}
+                />
+              </Form.Group>
+            </Form.Row>
+            <div className="d-flex justify-content-end">
+              <ModalCloseButton className="btntpz btn-text btn-tertiary">Cancel</ModalCloseButton>
```

Comment:
This button needs some space

```suggestion
              <ModalCloseButton className="btntpz btn-text btn-tertiary mr-3">Cancel</ModalCloseButton>
```